app/controllers/api/v1/search_results_controller.rb

```diff
@@ -27,6 +27,15 @@ def index
         render_success(serializer: search_result_serializer, status: :ok)
       end
 
+      def show
+        search_result = SearchResult.find(params[:id])
+        search_result_serializer = SearchResultDetailsSerializer.new(search_result)
```

Comment:
Since the model's name is `SearchResult`, then it should `SearchResultSerializer`. For the create method, we missed the fact that it should not be plural SearchResult**s**Serializer. For lists, the default serializer can be used with an extra option for collection (see https://github.com/jsonapi-serializer/jsonapi-serializer#collection-serialization)